Apps/WebClient/src/ClientApp/src/views/loginCallback.vue

```diff
@@ -35,22 +35,30 @@ export default class LoginCallbackView extends Vue {
                 this.logger.debug(
                     `oidcSignInCallback for user: ${JSON.stringify(this.user)}`
                 );
-                this.checkRegistration({ hdid: this.user.hdid }).then(() => {
```

Comment:
Not sure this code is necessary. It only goes to this route if we logged in an keycloak redirects back to this page. In the case of IDIR, I don't think it would go through the keycloak redirection

Reply:
@FuriousLlama Do you mean that we should also call this.checkRegistration for idirLoggedIn scenario as if keycloak, it is not idir logged in right?

Reply:
Updated per our PR review